<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Vadim Litvinov</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TeamPlanets Engine</string>
  </property>
  <property name="locale">
   <locale language="C" country="AnyCountry"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="team_planets_engine::MapWidget" name="battleMap" native="true"/>
      <widget class="QSplitter" name="teamsSplitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="team1Layout">
         <item>
          <widget class="QLabel" name="team1Label">
           <property name="text">
            <string>Team &amp;1:</string>
           </property>
           <property name="buddy">
            <cstring>team1Table</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="team1Table"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="team2Layout">
         <item>
          <widget class="QLabel" name="team2Label">
           <property name="text">
            <string>Team &amp;2:</string>
           </property>
           <property name="buddy">
            <cstring>team2Table</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="team2Table"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="startBattleAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <addaction name="fileMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="startBattleAction">
   <property name="text">
    <string>&amp;Start battle...</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>--</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>team_planets_engine::MapWidget</class>
   <extends>QWidget</extends>
   <header>mapwidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
